### PR TITLE
fix: align the response output structure

### PR DESCRIPTION
**Pull request by rubylo718**
1. modify response output structure (getUser, getTweets)
2. add "&&" in dev script in package.json for better usage

### CODE REVIEW

**Review by ritachien — COMMENTED** [review]
Test file has defined the response structure. Let's align to that! 
**Error happends**
```json
{
    "status": "error",
    "message": "error message",
}
```
**Success request**
return an array or object in request body first layer

**Comment by rubylo718** [comment]
[檢查統一回傳資料格式](https://trello.com/c/5JoG1RpJ/38-%E6%AA%A2%E6%9F%A5%E7%B5%B1%E4%B8%80%E5%9B%9E%E5%82%B3%E8%B3%87%E6%96%99%E6%A0%BC%E5%BC%8F)

File: controllers/tweet-controller.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 const { Tweet, User } = require('../models')
+// const helpers = require('../_helpers')
```
Comment (ritachien) [inline controllers/tweet-controller.js]: The response is align to test files!
And...at line 2, what's the reason to include the helpers module but comment it out?

Comment (rubylo718) [reply controllers/tweet-controller.js]: Sorry that was something left while error finding when the test failed (but was pass in fact lol). Deleted.

**Review by ritachien — APPROVED** [review]
Everything looks good!

**Pull request merged**